lib/tramway/forms/properties.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+module Tramway
+  module Forms
+    # This is `properties`. The main feature of Tramway Form
+    module Properties
+      # A collection of methods that would be using in users forms
+      module ClassMethods
+        def property(attribute)
+          @properties << attribute
+
+          delegate attribute, to: :object
+        end
+
+        def properties(*attributes)
+          attributes.any? ? __set_properties(attributes) : __properties
+        end
+
+        def __set_properties(attributes)
+          attributes.each do |attribute|
+            property(attribute)
+          end
+        end
+
+        def __properties
+          (__ancestor_properties + @properties).uniq
+        end
+
+        # :reek:ManualDispatch { enabled: false }
+        def __ancestor_properties(klass = superclass)
+          superklass = klass.superclass
+
+          return [] unless superklass.respond_to?(:properties)
+
+          klass.properties + __ancestor_properties(superklass)
+        end
+
+        # :reek:UtilityFunction { enabled: false }
+        def __initialize_properties(subclass)
+          subclass.instance_variable_set(:@properties, [])
+        end
+      end
+
+      def self.included(base)
+        base.extend ClassMethods
+        base.class_eval do
+          @properties = []
+        end
+      end
```

Comment:
It is actual code? Looks like experiment

Reply:
Yeah! This is an experiment. Thanks! Looks like I was tired a bit :slightly_smiling_face: